src/common/state/table.h

```diff
@@ -44,6 +44,11 @@ typedef struct {
   table_fail_callback fail_callback;
 } RetryInfo;
 
+static const RetryInfo heartbeat_retry = {
```

Comment:
We aren't using this struct initialization anywhere else in the code base (because normally it doesn't compile in C++, I have no idea why it's compiling here). 

E.g., http://stackoverflow.com/questions/5790534/static-structure-initialization-with-tags-in-c or http://stackoverflow.com/questions/11516657/c-structure-initialization. Obviously it's compiling so I'm mistaken about something, but it seems best to avoid this.

Can we just define it in `plasma_manager_send_heartbeat`? Or better yet, just pass in `NULL` to use the default retry info?